src/Datadog.Trace.ClrProfiler.Managed/Integrations/Testing/XUnitIntegration.cs

```diff
@@ -404,6 +404,7 @@ private static Scope CreateScope(object testSdk)
                 span.Type = SpanTypes.Test;
                 span.SetTraceSamplingPriority(SamplingPriority.AutoKeep);
                 span.ResourceName = $"{testSuite}.{testName}";
+                span.SetTag(Tags.Origin, TestTags.CIAppTestOriginName);
```

Comment:
I see that this tag is only getting added to the test integrations. If test cases generate automatic instrumentation (http client / db), do those need to have the tag too?

Reply:
is not required, in the current behaviour child spans always copy the SpanContext of the parent span.

This call has the parent as argument: https://github.com/DataDog/dd-trace-dotnet/blob/master/src/Datadog.Trace/Tracer.cs#L404

And that automatically will copy the origin to the new spancontext: https://github.com/DataDog/dd-trace-dotnet/blob/master/src/Datadog.Trace/SpanContext.cs#L62-L65

Reply:
Makes sense, thanks!